### PR TITLE
en: update doc for alert and monitor

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

- Update doc for alert and monitor.
- Fix a typo in `monitor-using-tidbmonitor.md`.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)
- [x] v1.1
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/36
- Other reference link(s):

### CODE REVIEW

File: en/monitor-a-tidb-cluster.md [hunk 1]
```diff
@@ -90,11 +90,16 @@ It is recommended to deploy a host monitoring system via [Prometheus Operator](h
...
 
-If you already have an available AlertManager service in your existing infrastructure, you can modify `monitor.prometheus.alertmanagerURL` in the `values.yaml` file and configure its address for use by Prometheus; if there is no AlertManager service available, or if you want to deploy a separate set of services, you can refer to [Prometheus official document](https://github.com/prometheus/alertmanager).
+- If you already have an available AlertManager service in your existing infrastructure, you can modify `spec.alertmanagerURL` and configure its address for use by Prometheus. For details, refer to [Set kube-prometheus and AlertManager](monitor-using-tidbmonitor.md#set-kube-prometheus-and-alertmanager).
+
+- If there is no AlertManager service available, or if you want to deploy a separate set of services, you can refer to [Prometheus official document](https://github.com/prometheus/alertmanager).
```
Comment (DanielZhangQD) [inline en/monitor-a-tidb-cluster.md]:
```suggestion
- If there is no AlertManager service available, or if you want to deploy a separate AlertManager service, you can refer to [Prometheus official document](https://github.com/prometheus/alertmanager).
```

File: en/monitor-a-tidb-cluster.md [hunk 2]
```diff
@@ -90,11 +90,16 @@ It is recommended to deploy a host monitoring system via [Prometheus Operator](h
...
 
 The default Prometheus and alert configuration do not support sending alert messages. To send an alert message, you can integrate Prometheus with any tool that supports Prometheus alerts. It is recommended to manage and send alert messages via [AlertManager](https://prometheus.io/docs/alerting/alertmanager/).
 
-If you already have an available AlertManager service in your existing infrastructure, you can modify `monitor.prometheus.alertmanagerURL` in the `values.yaml` file and configure its address for use by Prometheus; if there is no AlertManager service available, or if you want to deploy a separate set of services, you can refer to [Prometheus official document](https://github.com/prometheus/alertmanager).
+- If you already have an available AlertManager service in your existing infrastructure, you can modify `spec.alertmanagerURL` and configure its address for use by Prometheus. For details, refer to [Set kube-prometheus and AlertManager](monitor-using-tidbmonitor.md#set-kube-prometheus-and-alertmanager).
```
Comment (DanielZhangQD) [inline en/monitor-a-tidb-cluster.md]:
```suggestion
- If you already have an available AlertManager service in your existing infrastructure, you can set the value of `spec.alertmanagerURL` to the address of `AlertManager`, which will be used by Prometheus. For details, refer to [Set kube-prometheus and AlertManager](monitor-using-tidbmonitor.md#set-kube-prometheus-and-alertmanager).
```

Comment (ran-huang) [inline en/monitor-a-tidb-cluster.md]:
```suggestion
- If you already have an available AlertManager service in your existing infrastructure, you can set the value of `spec.alertmanagerURL` to the address of `AlertManager`, which will be used by Prometheus. For details, refer to [Set kube-prometheus and AlertManager](monitor-using-tidbmonitor.md#set-kube-prometheus-and-alertmanager).
```

File: en/monitor-a-tidb-cluster.md [hunk 3]
```diff
@@ -90,11 +90,16 @@ It is recommended to deploy a host monitoring system via [Prometheus Operator](h
...
 
-If you already have an available AlertManager service in your existing infrastructure, you can modify `monitor.prometheus.alertmanagerURL` in the `values.yaml` file and configure its address for use by Prometheus; if there is no AlertManager service available, or if you want to deploy a separate set of services, you can refer to [Prometheus official document](https://github.com/prometheus/alertmanager).
+- If you already have an available AlertManager service in your existing infrastructure, you can set the value of `spec.alertmanagerURL` to the address of `AlertManager`, which will be used by Prometheus. For details, refer to [Set kube-prometheus and AlertManager](monitor-using-tidbmonitor.md#set-kube-prometheus-and-alertmanager).
+
+- If there is no AlertManager service available, or if you want to deploy a separate AlertManager service, you can refer to [Prometheus official document](https://github.com/prometheus/alertmanager).
```
Comment (lilin90) [inline en/monitor-a-tidb-cluster.md]:
```suggestion
- If no AlertManager service is available, or if you want to deploy a separate AlertManager service, refer to the [Prometheus official document](https://github.com/prometheus/alertmanager).
```

**Comment by ran-huang** [comment]
@lilin90 PTAL again. Thanks!

**Pull request merged**